common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -441,6 +444,66 @@ public void run() {
         });
     }
 
+    @Override
+    public String sign(@NonNull final String input) throws ClientException {
+        // TODO Sign how???
+        // RS256? SHA256withECDSA?
+        return null;
+    }
+
+    @Override
+    public String getPublicKey(@NonNull final PublicKeyFormat format) throws ClientException {
+        if (X_509_SubjectPublicKeyInfo_ASN_1 == format) {
+            return getX509SubjectPublicKeyInfo();
+        } else if (PKCS1_RSAPublicKey == format) {
+            return getPkcs1RsaPublicKey();
+        } else {
+            throw new ClientException(UNKNOWN_EXPORT_FORMAT);
+        }
```

Comment:
If this is an enum, you could use a switch. Potential benefits, some static analysis tools will complain at you if you're not handling all of the potential enumerated values correctly.

Also, if you've got an unknown format, it would be really nice to have logged what the value actually was.

Reply:
8d10b1561f3ea65a6f3a3e6dc8f5da638c9ba86f, e78cf6b7b6a14598b1e39a107815a6aa62d7fc92